TAF/testScenarios/functionalTest/V2-API/app-service/trigger/POST-positive.robot

```diff
@@ -58,6 +58,7 @@ TriggerPOST005 - Trigger pipeline (JSON-ZLIB)
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
 
 TriggerPOST006 - Trigger pipeline (JSON-ZLIB-AES)
+    [Tags]  Skipped
```

Comment:
comment the reason of skipping this test

Reply:
Updated.